src/tab/tabs.ts

```diff
@@ -77,18 +77,28 @@ export class RemixTabs extends LitElement {
     if (index !== -1) {
       this.tabs = [...this.tabs.slice(0, index), ...this.tabs.slice(index + 1, this.tabs.length)];
     }
-    // send message to the parent to remove one tab and update the property
+    this.dispatchEvent(id)
+  }
+
+  /** activate a specific tab from the list */
+  public activateTab({ detail: id }: CustomEvent) {
+    const index = this.tabs.findIndex(tab => tab.id === id)
```

Comment:
I think you don't want to set the active tab manually here. You should rather having a property `active` to linked to the parent. 
Something like : 
```typescript
@property({ type: String }) active: string; 
```

Reply:
sure, it's there